src/main/java/Duke.java

```diff
@@ -1,10 +1,120 @@
+
+import java.util.Scanner;
+
 public class Duke {
+
+    public static String getInputType(String line) {
+        String breakLine[] = line.split(" ", 2);
+        return breakLine[0];
+    }
+
+    public static String getInputDetails(String line) {
+        String breakLine[] = line.split(" ", 2);
+        return breakLine[1];
+    }
+
+    public static void printTaskList(Task[] tasks, int taskCount) {
+        System.out.println("\t_____________________");
+        // if there are no task
+        if (taskCount == 0) {
+            System.out.println("There are no tasks yet!");
```

Comment:
Conditionals placed on seperate lines and wrapped with curly braces, even if they are single statement ones. Good!